sunpy/coordinates/tests/test_transformations.py

```diff
@@ -5,7 +5,8 @@
 from astropy.coordinates import SkyCoord, get_body_barycentric
 from astropy.time import Time
 
-from sunpy.coordinates import Helioprojective, HeliographicStonyhurst, HeliographicCarrington, get_sun_L0
+from sunpy.coordinates import Helioprojective, HeliographicStonyhurst,  
+                            HeliographicCarrington, get_sun_L0
```

Comment:
This new line for the imports break. You would need to move it inline.

Reply:
They should be with parenthesis (preferred) or `\`:
```python
from sunpy.coordinates import (Helioprojective, HeliographicStonyhurst,  
                               HeliographicCarrington, get_sun_L0)
```

```python
from sunpy.coordinates import Helioprojective, HeliographicStonyhurst, \
                              HeliographicCarrington, get_sun_L0
```